server/utils/validation/clientValidation.js

```diff
@@ -0,0 +1,24 @@
+const yup = require('yup');
+
+const schema = yup.object().shape({
+  fullName: yup.string().trim().required(),
+  email: yup.string().email().trim(),
+  mobileNumber: yup.string().trim(),
+  password: yup
+    .string()
+    .matches(
+      /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/,
+      'Must Contain 8 Characters, One Uppercase, One Lowercase, One Number and one special case Character'
+    )
+    .required(),
+  passwordConfirmation: yup.mixed().oneOf([yup.ref('password')]),
+  bankAccounts: yup.number().required(),
+  mainBank: yup.number().required(),
+  balance: yup.object().required(),
+  avatar: yup.string().url().trim(),
+  accepted: yup.boolean(),
```

Comment:
why `accepted` here ?

Reply:
I guess we want it if we want to send code for a client when he signup with mobile number 
@Asmaathabet

Reply:
I will delete it and we can use another way to check client code for his mobile
@Asmaathabet